src/posix.mak

```diff
@@ -2,7 +2,7 @@
 # Important variables:
 # --------------------
 #
-# HOST_CSS:				Host C++ compiler to use (g++,clang++)
+# HOST_CXX:				Host C++ compiler to use (g++,clang++)
```

Comment:
While you're at it, please fix the excessive spacing between the variable and the description so it lines up with the others.

Reply:
Those were tabs -> I "retabbed" them with spaces.